crates/router/src/connector/aci/transformers.rs

```diff
@@ -59,6 +71,162 @@ pub enum PaymentDetails {
     BankRedirect(Box<BankRedirectionPMData>),
     Wallet(Box<WalletPMData>),
     Klarna,
+    Mandate,
+}
+
+impl TryFrom<&api_models::payments::WalletData> for PaymentDetails {
+    type Error = Error;
+    fn try_from(wallet_data: &api_models::payments::WalletData) -> Result<Self, Self::Error> {
+        let payment_data = match wallet_data {
+            api_models::payments::WalletData::MbWay(data) => Self::Wallet(Box::new(WalletPMData {
+                payment_brand: PaymentBrand::Mbway,
+                account_id: Some(data.telephone_number.clone()),
```

Comment:
why account_id is mapped with telephone_number?

Reply:
For MBway, the account Id is accepting in the telephone number format i.e telephone number is the account id for MBway in ACI